### PR TITLE
fix for Ammonite does not properly backtick a value #663

### PR DESCRIPTION
**Pull request by justinhj**
works

### CODE REVIEW

**Comment by lihaoyi** [comment]
Looks good. Three  things to do before landing:

- Add something to the changelog here https://github.com/lihaoyi/Ammonite/blob/master/readme/Footer.scalatex#L1400

- Add this case to the unit test that's meant to test these case (https://github.com/lihaoyi/Ammonite/blob/master/amm/src/test/scala/ammonite/session/EvaluatorTests.scala#L289), make sure the old code fails your updated test, and the new code passes

- Also add a fix for operators containing `//`, which currently don't work for presumably the same reason:

```scala
@ val `//` = 1
cmd2.sc:77: ')' expected but '}' found.
      ) }
        ^
Compilation Failed
```

**Comment by justinhj** [comment]
Cool will address these issues

File: amm/src/test/scala/ammonite/session/EvaluatorTests.scala [hunk 1]
```diff
@@ -287,6 +287,37 @@ object EvaluatorTests extends TestSuite{
...
+
+        @ `Something then //`
+        res14: Int = 123
+
+        @ import ` `.{` ` => `// and then`}
+
+        @ `// and then`
+        res16: Int = 123
+
+        @ import ` `.{` ` => `/*`}
```
Comment (lihaoyi) [inline amm/src/test/scala/ammonite/session/EvaluatorTests.scala]: Could you convert one of these to 

```scala
val `/*` = 123
```

Just to make sure that works too

**Comment by justinhj** [comment]
I have tests for `/*` and `*/`

**Comment by lihaoyi** [comment]
Yeah I know you have tests for `/*`, I want you to also explicitly test the 

```scala
val `/*` = ...
```

syntax because it fails with a different error than the `import` command, and we should make sure that the `val` assignment works

**Comment by justinhj** [comment]
Ah i see, will do

**Comment by lihaoyi** [comment]
Looks good, thanks!

**Comment by lihaoyi** [comment]
Fixed in https://github.com/lihaoyi/Ammonite/issues/663

**Pull request merged**